### ISSUE TITLE
[BUG]: GS: Memory Allocation Failure

### ISSUE THREAD

**Issue by CristobalManga** [PCSX2/pcsx2#6254]
### Describe the Bug

When opening games like MGS3, MGS2 or Black, I get an error and the game turns black but still plays in the background.
And the error that comes out is this: GS: Memory allocation error.
![image](https://user-images.githubusercontent.com/78171867/170845042-f7a21d2e-0d3d-4c8f-9785-85ee56a930c6.png)

### Reproduction Steps

In this case it happens to me when I open MGS3, the game worked correctly for me in PCSX2 32BITS, but in these new 64-bit updates the error began to appear in these games, in other games like mortal kombat shaolinks monks, I recognize that I noticeably improved the speed with this 64-bit version.

### Expected Behavior

Well, the game should work with older versions. I'm running the game in native resolution and tried toggling the setting on and off, but the game remains the same.

### PCSX2 Revision

V1.7.2825

### Operating System

Windows 11

### CPU

I3-4160

### GPU

integrada Intel 4400

### GS Settings

![image](https://user-images.githubusercontent.com/78171867/170845153-ec52b5a8-54dc-44d9-ba07-3e280ce19911.png)

**Comment by refractionpcsx2** [comment]
can you try #6251 and see if that helps at all?

**Comment by CristobalManga** [comment]
> can you try #6251 and see if that helps at all?
> 
> link directly to the builds: https://github.com/PCSX2/pcsx2/actions/runs/2402215701
> 
> Edit: Actually I'm just gonna merge it so try updating to 2827 when it's built

I already tried it with the new update, but The problem still persists.

![image](https://user-images.githubusercontent.com/78171867/170845852-884abdae-9268-4778-adad-5ec913fc8460.png)

**Comment by refractionpcsx2** [comment]
tbh I'm not sure Windows 11 is even supposed to support your CPU, nevermind your iGPU. If I were you I'd see if you can update your drivers, or drop back down to windows 10.

It's more likely your GPU drivers (or lackthereof) screwing up, rather than what version of PCSX2 you're using.  I would suggest if you need further help you use our support system, as this is an isolated case and I suspect your setup is the issue.

**Comment by Nerboruto** [comment]
it happened to me on windows 10 too.
some games give this error when playing a cutscene es.. metal slug 3 or Ace Combat: The Belkan War
Ace Combat: Squadron Leader works perfectly without this error.
I have tried various backends and both the qt and vx version.
on qt a few times I managed to get out of the error by pressing esc and switching to the overlay and returning to the game.
on wx I think it can not be done. 
resolution is irrelevant it also does the native one.

[emuLog.txt](https://github.com/PCSX2/pcsx2/files/9564799/emuLog.txt)
